### PR TITLE
Add fluentd and fluentbit chart

### PR DESCRIPTION
**Pull request by pipo02mix**
Add fluentbit plus fluentd to let customers send logs (ssh,k8s audit and containers to S3)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,32 @@
 
 # Logging App
 
-Logging app is a proposal managed app to help customer to forward logs to AWS CLoudWatch or S3. 
+Logging app is a proposal managed app to help customer to forward logs to AWS CLoudWatch or S3. It contains two parts: fluentbit as a small collector running as daemonset in the clusters and forwarding the logs to fluentd which actually takes care of persisting this log in a third party service like S3 or CloudWatch.
```
Comment (rossf7) [inline README.md]:
```suggestion
Logging app is a proposed managed app to help customers forward logs to AWS CloudWatch or S3. It contains two parts: fluentbit is a small collector running as daemonset in the clusters and forwarding the logs to fluentd which actually takes care of persisting the logs to a third party service like S3 or CloudWatch.
```

File: helm/logging-app-chart/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+name: logging-app-chart
+version: 1.0.0-[[ .SHA ]]
```
Comment (rossf7) [inline helm/logging-app-chart/Chart.yaml]:
```suggestion
version: [[ .Version ]]
```

This has changed with App Catalog so I think it should be this.

File: helm/logging-app-chart/templates/fluentbit/daemonset.yaml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      serviceAccountName: {{ .Values.fluentbit.name }}
+      securityContext:
+        runAsUser: {{ .Values.fluentbit.userID }}
+        runAsGroup: {{ .Values.fluentbit.groupID }}
+      containers:
+      - name: {{ .Values.fluentbit.name }}
+        image: quay.io/giantswarm/fluent-bit:0.14.6
```
Comment (rossf7) [inline helm/logging-app-chart/templates/fluentbit/daemonset.yaml]: Should the images be templated for the registry etc?

Like in https://github.com/giantswarm/fmt/blob/master/helm/image_templating.md.

Comment (pipo02mix) [reply helm/logging-app-chart/templates/fluentbit/daemonset.yaml]: yeah good idea

File: helm/logging-app-chart/templates/fluentbit/daemonset.yaml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      serviceAccountName: {{ .Values.fluentbit.name }}
+      securityContext:
+        runAsUser: {{ .Values.fluentbit.userID }}
+        runAsGroup: {{ .Values.fluentbit.groupID }}
+      containers:
+      - name: {{ .Values.fluentbit.name }}
+        image: {{ .Values.fluentbit.image.registry }}/{{ .Values.fluentbit.image.name }}:{{ .Values.fluentbit.image.tag }}
```
Comment (rossf7) [inline helm/logging-app-chart/templates/fluentbit/daemonset.yaml]:
```suggestion
        image: {{ .Values.image.registry }}/{{ .Values.fluentbit.image.name }}:{{ .Values.fluentbit.image.tag }}
```

All the images should use `image.registry` as this is the only value we set in the app catalog configmap,

Comment (pipo02mix) [reply helm/logging-app-chart/templates/fluentbit/daemonset.yaml]: ahh ok got it

Comment (pipo02mix) [reply helm/logging-app-chart/templates/fluentbit/daemonset.yaml]: but then I need to remove the image from child values parameter, right?

Comment (pipo02mix) [reply helm/logging-app-chart/templates/fluentbit/daemonset.yaml]: and image to the top values yaml?

Comment (rossf7) [reply helm/logging-app-chart/templates/fluentbit/daemonset.yaml]: Its just image.registry this applies to.

I'm fine with `.Values.fluentbit.image.name` unless you think having all the images under `.Values.image` is better?

**Pull request merged**